### ISSUE TITLE
The score is not labeled 🏷

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#253]
The score is not readable by a screen reader. It should be labeled so that people know what the score of the website is.

<img width="1680" alt="score" src="https://user-images.githubusercontent.com/81969/43590181-698b338c-963e-11e8-84c0-8c38016e6f3d.png">

**Comment by andrey-p** [comment]
Fixed in #278.
